packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-tile.tsx

```diff
@@ -0,0 +1,393 @@
+import {
+  Component,
+  Prop,
+  h,
+  Element,
+  Host,
+  State,
+  Fragment,
+} from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import {
+  OdsChipSize,
+  OdsChipVariant,
+  OdsHTMLAnchorElementTarget,
+  OdsIconName,
+  OdsIconSize,
+} from '@ovhcloud/ods-core';
+import { HTMLStencilElement } from '@stencil/core/internal';
+import {
+  getTranslation,
+  fetchLocaleStringsForComponent,
+} from '../../../../utils/translation.utils';
+import formatDate from '../../../../utils/date.utils';
```

Comment:
This should be in a package
We can't import things outside of the super-component directory in the production code or it will be duplicated